src/share/poudriere/common.sh

```diff
@@ -2595,7 +2595,12 @@ jail_start() {
 
 	# May already be set for pkgclean
 	: ${PACKAGES:=${POUDRIERE_DATA}/packages/${MASTERNAME}}
-	: ${THIN_PACKAGES:=${POUDRIERE_DATA}/packages/${MASTERNAME}-thin}
+	if [ ${SMALL_REPO} -eq 1 ]; then
+		THIN_EXT=-small
```

Comment:
It's confusing that the options are minimal and medium, but the extensions are -thin and -small. At a minimum the documentation should mention this. Ideally things should lineup throughout.

Reply:
yes but minimal and medium are bad names, think and small are nice, but we can't use proper switches for them, I am open to any suggestion for renaming that